src/firejail/profile.c

```diff
@@ -938,6 +938,15 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 	if (strcmp(ptr, "apparmor") == 0) {
 #ifdef HAVE_APPARMOR
 		arg_apparmor = 1;
+		strcpy(apparmor_profile,"firejail-default");
+#endif
+		return 0;
+	}
+	
+	if (strncmp(ptr, "apparmor ",9) == 0) {
+#ifdef HAVE_APPARMOR
+		arg_apparmor = 1;
+		strcpy(apparmor_profile,ptr+9);
```

Comment:
`apparmor_profile` is `char[30]`, but here you are copying an arbitrary amount of bytes into it. please use `strncpy` (but please keep in mind that it might not null-terminate).  

the same apllied to `main.c`.

Reply:
btw very surprised that neither cppcheck, nor scan-build, nor CodeQL complain about that...

Reply:
> btw very surprised that neither cppcheck, nor scan-build, nor CodeQL complain about that...

Die someone startend the CI checks?

Reply:
Fixed -- now no more than 29 bytes are copied, and the null is placed to the end of the string. Force-pushed to prevent the unsafe version of the commit from being used anywhere.

Reply:
> Die someone startend the CI checks?

I have started them yesterday, but they didn't find anything. That's why I upgraded the CI runners in the other MR.
But weird that I now have to approve the CI again...